dashboards-observability/common/query_manager/ast/expression/span.ts

```diff
@@ -3,26 +3,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { CUSTOM_LABEL } from '../../../../common/constants/explorer';
 import { PPLNode } from '../node';
 
 export class Span extends PPLNode {
   constructor(
     name: string,
     children: Array<PPLNode>,
     private spanExpression: PPLNode,
-    private alias: string
+    private customLabel: string
   ) {
     super(name, children);
   }
 
   getTokens() {
     return {
       span_expression: this.spanExpression.getTokens(),
-      alias: this.alias,
+      [CUSTOM_LABEL]: this[CUSTOM_LABEL],
```

Comment:
since it is inside class, we can have` this.customLabel `here.

Reply:
done